code/player/Player.cs

```diff
@@ -141,6 +141,11 @@ public override void Simulate(IClient cl)
             ThirdPersonCamera = !ThirdPersonCamera;
         }
 
+        if (Input.Pressed("swap_shoulder"))
```

Comment:
`swap_shoulder` is different than `swap` which is found in `.addon`

Reply:
Forgot to push the .addon file